.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
Can we mention something here about having either a repo or an MVCE?

Reply:
Unfortunately most Cucumber users have a limited amount of experience with either. I'm afraid that by asking for anything specific will just get us nothing at all instead.